db/structure.sql

```diff
@@ -198,7 +198,8 @@ CREATE TABLE public.prosecution_case_defendant_offences (
     status_date timestamp without time zone,
     effective_start_date timestamp without time zone,
     effective_end_date timestamp without time zone,
-    defence_organisation json
+    defence_organisation json,
+    application_type character varying
```

Comment:
Just asking should we put some limits on this, for example: 
`character varying(255)`